tests/test_replication_key.py

```diff
@@ -3,76 +3,59 @@
 import json
 
 import pendulum
-import pytest
+import sqlalchemy
 from faker import Faker
-from singer_sdk.testing import get_standard_tap_tests
+from singer_sdk.testing.templates import TapTestTemplate
 from sqlalchemy import Column, DateTime, Integer, MetaData, String, Table
 
 from tap_postgres.tap import TapPostgres
 
+TABLE_NAME = "test_replication_key"
 SAMPLE_CONFIG = {
     "start_date": pendulum.datetime(2022, 11, 1).to_iso8601_string(),
     "sqlalchemy_url": "postgresql://postgres:postgres@localhost:5432/postgres",
 }
 
 
-# Run standard built-in tap tests from the SDK:
-def test_standard_tap_tests():
-    """Run standard tap tests from the SDK."""
-    tests = get_standard_tap_tests(TapPostgres, config=SAMPLE_CONFIG)
-    for test in tests:
-        test()
+def setup_test_table(table_name, sqlalchemy_url):
+    """setup any state specific to the execution of the given module."""
+    engine = sqlalchemy.create_engine(sqlalchemy_url)
+    fake = Faker()
 
-
-@pytest.fixture()
-def sqlalchemy_connection(tap):
```

Comment:
Note to myself to run tests locally, as this wasn't working without these override for me. As long as test pass we should be good

Reply:
Tests worked great for me locally! Super fast